jxl/src/frame/modular.rs

```diff
@@ -573,6 +582,18 @@ pub fn decode_vardct_lf(
         br,
     )?;
     // TODO(szabadka): Generate the f32 pixels of the LF image.
```

Comment:
Is this comment still true?

(as a side note, we might in the future want to avoid creating a full LF image)